job/common/src/main/java/alluxio/job/wire/JobWorkerHealth.java

```diff
@@ -84,7 +94,8 @@ public String getHostname() {
    */
   public alluxio.grpc.JobWorkerHealth toProto() {
     alluxio.grpc.JobWorkerHealth.Builder builder = alluxio.grpc.JobWorkerHealth.newBuilder()
-        .setWorkerId(mWorkerId).addAllLoadAverage(mLoadAverage).setLastUpdated(mLastUpdated);
+        .setWorkerId(mWorkerId).addAllLoadAverage(mLoadAverage).setTaskPoolSize(mTaskPoolSize)
```

Comment:
is there a unittest for this method?

Reply:
There is just an integration test for this. Let me add a unit test